File: Restaurant_advisor/Restaurant_Advisor_API/app/Restaurant.php
<?php

namespace App;

use Illuminate\Database\Eloquent\Model;
use Illuminate\Support\Facades\DB;

class Restaurant extends Model
{

  protected $table = 'restaurant';

    public function menu()
    {
      return $this->hasMany(Menu::class);
    }

    function getrestaurant()
    {
        $Restaurant = DB::table('restaurant')->get();
        return $Restaurant;
    }

    function getrestaurantb($id)
    {
        $Restaurant = DB::table('restaurant')
            ->where('id', $id)
            ->get();
        return $Restaurant;
    }

    function createrestaurant($description, $grade, $localization, $phone_number, $website, $hour)
    {
        DB::table('restaurant')->insert(
            [
              'name' => $name,
              'description' => $description,
              'grade' => $grade,
              'localization' => $localization,
              'phone_number' => $phone_number,
              'website' => $website,
              'hour' => $hour
            ]
        );
    }

    function updaterestaurant($id, $description, $grade, $localization, $phone_number, $website, $hour)
    {
        DB::table('restaurant')->where('id', $id)
            ->update(
              [
                'name' => $name,
                'description' => $description,
                'grade' => $grade,
                'localization' => $localization,
                'phone_number' => $phone_number,
                'website' => $website,
                'hour' => $hour
              ]
        );
    }


    function removerestaurant($id)
    {
        DB::table('restaurant')->where('id', $id)->delete();
    }
}

File: Restaurant_advisor/Restaurant_Advisor_Android_APP/Liste les menus/VolleyMySQLExample/app/src/main/java/net/simplifiedlearning/volleymysqlexample/Restaurant.java
package net.simplifiedlearning.volleymysqlexample;


public class Restaurant {
    private int id;
    private String name;
    private String description;
    private double grade;
    private String localization;
    private String phone_number;
    private String website;

    public Restaurant(int id, String name, String description, double grade, String localization, String phone_number, String website) {
        this.id = id;
        this.name = name;
        this.description = description;
        this.grade = grade;
        this.localization = localization;
        this.phone_number = phone_number;
        this.website = website;
    }
    public int getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public String getDescription() {
        return description;
    }

    public double getGrade() {
        return grade;
    }

    public String getLocalization() {
        return localization;
    }

    public String getPhone_number() {
        return phone_number;
    }

    public String getWebsite() {
        return website;
    }
}
